src/frontend/magnify/src/i18n/Messages/formLabelMessages.ts

```diff
@@ -0,0 +1,9 @@
+import { defineMessages } from 'react-intl';
+
+export const formLabelMessages = defineMessages({
```

Comment:
This message is currently not used. You should use it everywhere where this string is used.
Furthermore, you could add other common field here.